src/db/tables/breaches.js

```diff
@@ -60,8 +61,8 @@ async function upsertBreaches(hibpBreaches) {
           modified_date: breach.ModifiedDate,
           pwn_count: breach.PwnCount,
           description: breach.Description,
-          logo_path: breach.LogoPath,
-          data_classes: breach.DataClasses,
+          logo_path: /** @type {RegExpExecArray} */(/[^/]*$/.exec(breach.LogoPath))[0],
```

Comment:
Do you know why this regex is necessary?

Reply:
Unfortunately I don't; I moved that over from https://github.com/mozilla/blurts-server/blame/642be732f7c8f45c290c7407ade85a395388f113/src/utils/hibp.js#L115, but there's no clarification there.